hieradata/development.yaml

```diff
@@ -103,11 +102,13 @@ govuk::apps::ckan::priority_worker_processes: '1'
 govuk::apps::ckan::bulk_worker_processes: '1'
 govuk::apps::collections_publisher::enable_procfile_worker: false
 govuk::apps::contacts::extra_aliases: ['contacts-admin']
+govuk::apps::content_data_admin::enabled: true
```

Comment:
Why's this added?

Reply:
This enables content data admin in development. The reason we need to set is here is because it is being set to false in hiera common.yaml to disable it in carrenza.